forestflow/model_p3d_arinyo.py

```diff
@@ -265,11 +265,12 @@ def Px_Mpc(self, z, k_par, pp):
             rperp (array-like): values (float) of separation in Mpc
             Px_per_kpar (array-like): values (float) of Px for each k parallel and rperp. Shape: (len(k_par), len(rperp)).
         """
-
-        rperp, Px_per_kpar = pcross.Px_Mpc(
-            k_par, self.P3D_Mpc, z, P3D_mode="pol", **{"pp": pp}
+        list_check = ["bias", "beta", "q1", "q2", "kvav", "av", "bv", "kp"]
```

Comment:
I would not hard code the parameters here. At some point, I trained a ForestFlow emulator without q2, and we may want to do it in the future, or add an additional parameter. I think the list of parameters should be defined somewhere else, but if not, I would give it as an input to the class and I would define a list like self.name_params = name_params, or something similar